.github/workflows/deploy-master.yml

```
@@ -15,22 +20,34 @@ jobs:
+          run_install: false
+
+      - name: Setup NodeJS ✨
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          cache: "pnpm"
+
       - name: Install project dependencies 📦
-        run: npm ci
+        run: pnpm install
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
```

actionlint (1.7.4)

property "cache-node-modules" is not defined in object type {}

(expression)